initial_pose: [0.0, 0.0, 0.0] # [x, y, theta]
transform_to_base: [[0.0, -0.01, 0.15], [0.0, 1.57, 0.0]] # ([x, y, z], [roll, pitch, yaw])
calibration_multiplier: 0.000096954 # pixel to meter transformation
image_topic: '/camera/camera/color/image_raw'
tf_frame_name: 'base_estimation'
show_flows: True # Default is False
